claude/skills/investigate-incident/references/probes.md

```
@@ -0,0 +1,180 @@
+```
+search_events(
+  organizationSlug=<org>, projectSlug=<project>, dataset='errors',
+  fields=['environment', 'count()'], sort='-count()', period='24h', limit=20
+)
+```
+
+### 3. 症状の時系列（インシデント調査の主用途）
+
+障害調査で欲しいのは issue 一覧ではなく **「いつ何が増えたか」**。時間軸で集計する:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)